TeamCode/src/main/java/org/firstinspires/ftc/teamcode/auton/Tests/MotorEncoderTest.java

```diff
@@ -0,0 +1,102 @@
+package org.firstinspires.ftc.teamcode.auton.Tests;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+import com.qualcomm.robotcore.hardware.DcMotor;
+
+import org.firstinspires.ftc.teamcode.subsystems.hardware.*;
+
+@Autonomous(name="Motor Encoder Test")
+
+public class MotorEncoderTest extends OpMode {
+    private float wheelEncoderPPR = 537.7f; // PPR
+    private int wheelDiameter = 96; // mm
+    private double mmPerEncoderTick = (360/wheelEncoderPPR)/360*(wheelDiameter*Math.PI); // 0.56089435511 mm
+    private double distance;
+    private float motorRunPower = 0.4f;
+    private float tolerance = 1;
+    // TODO: Figure out actual value of tolerance.
```

Comment:
Usually, you would want to have TODOs/comments above the line you are commenting on, but its fine.